_acs-aem-commons/features/redirect-manager/index.md

```diff
@@ -85,7 +85,7 @@ to replicate your redirects to the publish instances.
 
 ### Testing 
 
-In your publish instance navigate to http://localhost:4503/content/geometrixx/us/en/about-us.html which should return 
+Navigate to http://localhost:4503/content/geometrixx/us/en/about-us.html which should return 
```

Comment:
```/content/geometrixx```? This isn't shipped for quite some time now, can't we switch over completely to we-retail?

Reply:
@joerghoh it's only an example, but I renamed it for consistency sake.